### DISCUSSION TITLE
Position Popup at Bottom of Screen with Bottom-Aligned URL Bar and Navigation Toolbox

### DISCUSSION THREAD

**Discussion by ashish-kus** [soulhotel/FF-ULTIMA#177] [General]
Hi everyone,

I'm trying to design a popup that appears at the bottom of the screen rather than the default top position. My setup includes both the URL bar and navigation toolbox positioned at the bottom of the screen, so ideally, the popup would appear just above these elements, maintaining a bottom-aligned layout.

I've looked through the documentation and tried a few different approaches, but I haven't had any luck achieving this layout. Do you know if anyone has experience reversing the popup position or knows how to get it to display at the bottom?

Any guidance or suggestions would be appreciated. Thanks!

**Comment by soulhotel** [comment]
Are you talking about an already existing popup, specify which one you are talking about please, a picture might make it easier

**Reply by ashish-kus** [reply]
![image](https://github.com/user-attachments/assets/25a4dc66-b5da-4ba5-84e1-4a29b6bc660d)
![image](https://github.com/user-attachments/assets/8a24f5c5-6f84-422f-be59-7b63a0960132)


My bad, This type of popup.
